README.md

```diff
@@ -83,7 +83,7 @@ Or you can exclude your particular custom post type from the confirmation dialog
 );</pre>
 
 ### Is there any way to change the default dialogue message into something else? ###
-The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme’s functions.php:
+The message text in the publishing dialogue can be changed via PHP filter from a custom plugin or your theme's functions.php:
```

Comment:
We used an apostrophe (`’`) intentionally there. Any specific reason you would want to change it?